changelog.md

```diff
@@ -1,5 +1,17 @@
 # Change Log
 
+## 2023-07 - Version 2.3.0
+
+1. **Added functionality for initializing pre-simulation leaks** Users can now utilize n_init_days and n_init_leaks to initialize pre-simulation start leaks.
```

Comment:
A second sentence adding some more details here might be helpful to users.

Reply:
My thoughts are that the change log should be brief, key points. If users wanted more details, they should go look at the USER_MANUAL.md to find out more.